stravalib/client.py

```diff
@@ -1305,15 +1370,20 @@ def get_segment_efforts(
         )
 
     def explore_segments(
-        self, bounds, activity_type=None, min_cat=None, max_cat=None
-    ):
+        self,
+        bounds: tuple[float, float, float, float]
```

Comment:
I'm curious why this must be a tuple (instead of any indexable sequence (which I wouldn't know how to type, BTW)).

Reply:
The tuple allows to incorporate the fact that there is exactly for float in the sequence, so that's why I chose it. Note that the function with still work if call with the list but the mypy will complain.

For a general indexable sequence you can use `typing.Sequence[float]`